src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.FileReader;
+import core.basesyntax.dao.FileReaderImpl;
+import core.basesyntax.dao.FileWriter;
+import core.basesyntax.dao.FileWriterImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.OperationType;
+import core.basesyntax.service.FruitShopService;
+import core.basesyntax.strategy.BalanceOperationHandler;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.PurchaseOperationHandler;
+import core.basesyntax.strategy.ReturnOperationHandler;
+import core.basesyntax.strategy.SupplyOperationHandler;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final String SOURCE_PATH_NAME = "src/data/fruit_shop_transactions";
+    private static final String TARGET_PATH_NAME = "src/reports/report";
```

Comment:
Avoid hardcode in your solution. The path to the target file should be received as a parameter, not hardcoded.